.env.schema

```
@@ -28,3 +28,5 @@ NEXT_PUBLIC_KEYCLOAK_REALM=myrealm
 NEXT_PUBLIC_KEYCLOAK_CLIENT_ID=webclient
 KEYCLOAK_CLIENT_SECRET=client_secret
 NEXT_PUBLIC_KEYCLOAK_PERMISSION_AUDIENCE=song
+
```

Thinking this may be useful for future reference later
```suggestion
// Score file access is needed for IOBIO and jBrowse components. This variable is not needed in vanilla Stage or Arranger
```

---

Added